File: Fresh/Fresh.Model/AdminDashboardDto.cs
namespace Fresh.Model
{
    public class AdminDashboardDto
    {
        public int TotalProducts { get; set; }
        public int TotalClients { get; set; }
        public int TotalPurchases { get; set; }
        public decimal TotalPaid { get; set; }
        public decimal TotalDebt { get; set; }
        public List<PurchasedProductsDto> PurchasedProducts { get; set; } = new List<PurchasedProductsDto>();
        public List<MonthlyPurchaseDto> MonthlyPurchase { get; set; } = new List<MonthlyPurchaseDto>();
        public List<TopClientsDto> TopClients { get; set; } = new List<TopClientsDto>();
        public List<PaymentTypeDto> PaymentType { get; set; } = new List<PaymentTypeDto>();
    }
}

File: Fresh/UI/fresh_app/src/app/model/AdminDashboardDto.ts
import { MonthlyPurchaseDto } from "./MonthlyPurchaseDto";
import { PaymentType } from "./PaymentType";
import { PaymentTypeDto } from "./PaymentTypeDto";
import { PurchasedProductsDto } from "./PurchasedProductsDto";
import { TopClientsDto } from "./TopClientsDto";

export interface AdminDashboardDto {
    totalProducts: number;
    totalClients: number;
    totalPurchases: number;
    totalPaid: number;
    totalDebt:number;

    purchasedProducts: PurchasedProductsDto[];
    monthlyPurchase: MonthlyPurchaseDto[];
    topClients: TopClientsDto[];
    paymentType: PaymentTypeDto[];
}